app/main/views/g_cloud.py

```diff
@@ -615,9 +615,9 @@ def get_file_context(self, **kwargs):
         manifest_questions = get_questions_from_manifest_by_id(download_results_manifest)
         price_question = list(filter(lambda x: type(x) == Pricing, manifest_questions.values()))
 
-        locked_at = datetime.strptime(project['lockedAt'], DATETIME_FORMAT)
-        filename = '{}-{}-results'.format(locked_at.strftime('%Y-%m-%d'), inflection.parameterize(project['name']))
-        locked_at = utcdatetimeformat(locked_at)
+        utc_locked_at = datetime.strptime(project['lockedAt'], DATETIME_FORMAT)
+        filename = '{}-{}-results'.format(utc_locked_at.strftime('%Y-%m-%d'), inflection.parameterize(project['name']))
```

Comment:
In the edge case that the filename would be different depending on whether we are using local time or UTC, it seems a bit confusing for the user to use UTC for the date. Why not go with the local date for consistency?

Reply:
We did wonder about that and thought it would be unlikely the buyer would be ending a search between 12am-1am, but happy to change for consistency. I'm not 100% where this file gets used - is it just for the buyer's own reference?

Reply:
Yeah this is just the download CSV of the buyer's shortlisted services? Yes it's not uploaded again anywhere else, so the naming is just to help the buyer be organised.

Reply:
Cool. Hopefully they will never notice any change (unless they are/have been working very late!)